app/src/index.js

```diff
@@ -28,6 +30,38 @@ const handleSocketClose = () => {
   document.getElementById('stopRecordButton').disabled = true;
 };
 
+const getVideoCodecs = () => {
+  let params = new URLSearchParams(location.search.slice(1));
+  let videoCodec = params.get('videocodec')
+  if (!videoCodec) {
```

Comment:
What is the reason that H264 is the default codec?

Reply:
It's no enough reason to choose H264 the default but there should be ONE when the parameter is not given, maybe others such vp8/vp9